recipe/meta.yaml

```diff
@@ -33,8 +34,8 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - python
-    - boost
-    - boost-cpp
+    - libboost-python-devel
```

Comment:
```suggestion
```
According to the link check, we don't need the python bits of boost.

Reply:
Thanks, applied the change.